extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/SmallRyeOpenApiProcessor.java

```diff
@@ -543,6 +544,20 @@ private OASFilter getAutoSecurityFilter(List<SecurityInformationBuildItem> secur
         return null;
     }
 
+    private boolean hasEnabledAuthPermission(HttpBuildTimeConfig httpConfig,
```

Comment:
Hey Mike, does it matter which path is covered by a given policy ?

Reply:
I don't think it does, it seems like purpose of this method is to detect if any path is secured at all (either on Vert.x or JAX-RS level).

Reply:
@sberyozkin , I think it doesn't. The current annotation-based support will only check that there are REST endpoints protected by `@RolesAllowed` or `@Authenticated`. It also checks whether an existing OpenAPI `@SecurityRequirement` is present, so that part is missing from the configuration-only approach which is a possible gap.